src/main/java/gov/cms/mat/cql_elm_translation/service/EffectiveDataRequirementService.java

```diff
@@ -30,7 +27,7 @@ public class EffectiveDataRequirementService {
   private RequestData createDefaultRequestData(String cql) {
     return RequestData.builder()
         .cqlData(cql)
-        .showWarnings(false)
+        .errorSeverity(CqlCompilerException.ErrorSeverity.Error)
```

Comment:
I think this is used for export. should this be configurable?

Reply:
For now, We have future stories to handle what should be the error level. User will select from UI, then this selection will become dynamic